test/jdk/javax/swing/JTable/KeyBoardNavigation.java

```diff
@@ -224,13 +173,128 @@ public void setValueAt(Object aValue, int row, int column) {
     }
 
     public static void main(String[] args) throws Exception {
-        SwingUtilities.invokeAndWait(() -> {
-            try {
-                initTest();
-            } catch (Exception e) {
-                throw new RuntimeException(e);
-            }
-        });
-        passFailJFrame.awaitAndCheck();
+        String INSTRUCTIONS = """
+                Instructions to Test:
+                1. Refer the below keyboard navigation specs
+                 (referenced from bug report 4112270).
+                2. Check all combinations of navigational keys mentioned below
+                 and verifying each key combinations against the spec defined.
+                 If it does, press "pass", otherwise press "fail".
+
+                """;
+
+        INSTRUCTIONS += getOSSpecificInstructions();
+        PassFailJFrame.builder()
+                .instructions(INSTRUCTIONS)
+                .rows(30)
+                .columns(50)
+                .testUI(KeyBoardNavigation::initTest)
+                .testTimeOut(10000)
```

Comment:
It should be 10 only, `testTimeOut` takes the parameter in terms of **minute not seconds**.

Reply:
Yeah, updated.